.eslintrc

```diff
@@ -85,22 +85,29 @@
     "react/sort-comp": 2,
     "react/style-prop-object": 2,
     "react/void-dom-elements-no-children": 2,
-    "react/function-component-definition": [2, { "namedComponents": ["function-declaration", "arrow-function"] }],
+    "react/function-component-definition": [
+      2,
+      { "namedComponents": ["function-declaration", "arrow-function"] }
+    ],
     "react-hooks/rules-of-hooks": 2, // Checks rules of Hooks
     "react-hooks/exhaustive-deps": 1, // Checks effect dependencies
     "fp/no-mutating-methods": 1
   },
   "overrides": [
     {
       "files": ["**/*.ts", "**/*.tsx"],
-      "plugins": [
-        "@typescript-eslint"
-      ],
+      "plugins": ["@typescript-eslint"],
       "rules": {
         "prettier/prettier": "off",
-        "no-unused-vars": "off"
+        "@typescript-eslint/no-unused-vars": "error"
```

Comment:
It would be nice to settle on how we express rule levels, either `0|1|2` or `off|warning|error` and stick to it. Personal preference for the numerical version but up to you.

Reply:
No preference! let's do numbers!